### PR TITLE
Feature/registration forms

### PR DESCRIPTION
**Pull request by luisabellan**
**ATTENTION**: For it to work correctly the following PR has to be accepted first: https://github.com/Lambda-School-Labs/LabsPT12-Express_Groomer-TeamA-BE/pull/16

- Created customer registration form 

- Created groomer registration form

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -0,0 +1,3 @@
+REACT_APP_CLIENT_ID=0oalwp37fU2aV9UEG4x6
+REACT_APP_OKTA_ISSUER_URI=https://auth.lambdalabs.dev/oauth2/default
+REACT_APP_API_URI=https://labspt12-express-groomer-a-api.herokuapp.com/
```
Comment (coryortega) [inline .env.sample:3]: Need to remove .env.sample file

File: src/components/Forms/Register.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+      <div>
+        <h2>Are you a...</h2>
+        <Link to="/groomers">Groomer</Link>
+        <Link to="/customers">Customer</Link>
+      </div>
+    );
+  }
+}
+
+export default Register;
```
Comment (coryortega) [inline src/components/Forms/Register.js]: Not entirely necessary, but could be converted to functional component

File: src/components/Layouts/Navbar.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+          <Link className="nav-link" to="/register">
+            Register
+          </Link>
+        </div>
+      </nav>
+    );
+  }
+}
+
+export default Navbar;
```
Comment (coryortega) [inline src/components/Layouts/Navbar.js]: Navbar might not be necessary, now that William implemented an Ant design dashboard with a nav on the side

File: src/components/Forms/RegisterCustomer.js [hunk 4]
```diff
@@ -0,0 +1,306 @@
...
+
+  // const handleSubmit = (e) => {
+  //   e.preventDefault()
+  //   axios.post('/customers', customerRegister)
+  //     .then(function (response) {
+  //         console.log(response)
+  //     })
+  //     .catch(function (error) {
+  //         console.log(error)
+  //     })
```
Comment (coryortega) [inline src/components/Forms/RegisterCustomer.js]: While I think the comments are nice so Adella can see the changes made, we ultimately should delete them before merging to main

**Review by coryortega — CHANGES_REQUESTED** [review]
Nice job, just need to remove .env.sample file and also delete comments, refactor class component to functional component

File: .env.sample (hunk 1 above)
Comment (luisabellan) [reply .env.sample:3]: I can't remove it, apparently the tests requires a .env.sample to be there:
https://gyazo.com/53f3c8e490df89e3304b5439de236328
https://gyazo.com/f0dc2fe8bd39703aecc29d44ae5a1c73

File: src/components/Forms/Register.js (hunk 2 above)
Comment (luisabellan) [reply src/components/Forms/Register.js]: done :+1:

File: src/components/Layouts/Navbar.js (hunk 3 above)
Comment (luisabellan) [reply src/components/Layouts/Navbar.js]: removed

File: src/components/Forms/RegisterCustomer.js (hunk 4 above)
Comment (luisabellan) [reply src/components/Forms/RegisterCustomer.js]: done

**Comment by luisabellan** [comment]
> Nice job, just need to remove .env.sample file and also delete comments, refactor class component to functional component

test coverage requires a .env.sample file so it can recreate .env (cp .env.sample .env):
https://gyazo.com/f0dc2fe8bd39703aecc29d44ae5a1c73

**Review by luisabellan — COMMENTED** [review]
.env.sample must be present so it can be copied into .env to recreate it (cp .env.sample .env)

**Pull request closed without merging**